<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabTerminals"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTerminal">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="actionFile_New">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>New</string>
     </property>
     <property name="separatorsCollapsible">
      <bool>false</bool>
     </property>
     <addaction name="actionFile_New_Serial"/>
     <addaction name="actionFile_New_Socket"/>
    </widget>
    <addaction name="actionFile_New"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Open"/>
    <addaction name="actionFile_Save"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionControl_ResetTerminal"/>
    <addaction name="actionReset_buffer"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionControl_MakeBridge"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_ClearScreen"/>
    <addaction name="actionEdit_SelectAll"/>
    <addaction name="actionEdit_SelectScreen"/>
    <addaction name="actionEdit_Copy"/>
    <addaction name="actionEdit_Paste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_HomePage"/>
    <addaction name="actionHelp_Sources"/>
    <addaction name="separator"/>
    <addaction name="actionHelp_About"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionSetup_Terminal"/>
    <addaction name="actionSetup_Application"/>
   </widget>
   <addaction name="menuTerminal"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSetup"/>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile_New_Serial"/>
   <addaction name="actionFile_NewConsole"/>
   <addaction name="actionFile_NewSsh"/>
   <addaction name="actionFile_New_Socket"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSerial">
   <property name="text">
    <string>Serial</string>
   </property>
  </action>
  <action name="actionFile_New_Serial">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/new-terminal-serial.svg</normaloff>:/icon/resources/icon/new-terminal-serial.svg</iconset>
   </property>
   <property name="text">
    <string>Serial</string>
   </property>
   <property name="shortcut">
    <string>Alt+N, Alt+S</string>
   </property>
  </action>
  <action name="actionFile_NewConsole">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/new-terminal-console.svg</normaloff>:/icon/resources/icon/new-terminal-console.svg</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="shortcut">
    <string>Alt+N, Alt+C</string>
   </property>
  </action>
  <action name="actionFile_NewSsh">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/new-terminal-ssh.svg</normaloff>:/icon/resources/icon/new-terminal-ssh.svg</iconset>
   </property>
   <property name="text">
    <string>SSH</string>
   </property>
   <property name="shortcut">
    <string>Alt+N, Alt+H</string>
   </property>
  </action>
  <action name="actionFile_New_Socket">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/new-terminal-socket.svg</normaloff>:/icon/resources/icon/new-terminal-socket.svg</iconset>
   </property>
   <property name="text">
    <string>Socket</string>
   </property>
   <property name="shortcut">
    <string>Alt+N, Alt+K</string>
   </property>
  </action>
  <action name="actionFile_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTerminal_Intermediate">
   <property name="text">
    <string>Intermediate</string>
   </property>
  </action>
  <action name="actionFile_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionControl_ResetTerminal">
   <property name="text">
    <string>Reset terminal</string>
   </property>
  </action>
  <action name="actionControl_Configure">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEdit_SelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionEdit_ClearScreen">
   <property name="text">
    <string>Clear screen</string>
   </property>
  </action>
  <action name="actionHelp_Sources">
   <property name="text">
    <string>Sources</string>
   </property>
  </action>
  <action name="actionHelp_HomePage">
   <property name="text">
    <string>Home page</string>
   </property>
  </action>
  <action name="actionHelp_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEdit_Copy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionEdit_Paste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionEdit_SelectScreen">
   <property name="text">
    <string>Select screen</string>
   </property>
  </action>
  <action name="actionTerminal_Stdio">
   <property name="text">
    <string>Stdio</string>
   </property>
  </action>
  <action name="actionControl_MakeBridge">
   <property name="text">
    <string>Make bridge</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure Terminal</string>
   </property>
  </action>
  <action name="actionSetup_Application">
   <property name="text">
    <string>Application</string>
   </property>
  </action>
  <action name="actionSetup_Terminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/connected.svg</normaloff>:/icon/resources/icon/connected.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/resources/icon/disconnected.svg</normaloff>:/icon/resources/icon/disconnected.svg</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionReset_buffer">
   <property name="text">
    <string>Reset buffer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
